packages/cli/src/util/telemetry/index.ts

```diff
@@ -238,8 +239,9 @@ export class TelemetryEventStore {
       }
       const events = this.events.map(event => {
         delete event.sessionId;
-        const { eventTime, teamId, ...rest } = event;
-        return { event_time: eventTime, team_id: teamId, ...rest };
+        delete event.teamId;
```

Comment:
If I'm reading correct, the `team_id` value will only be sent for one metric in the events array - is that correct / intended?

Reply:
No, the `team_id` field will be set for every event in the batch of events we'll be sending. The main part of this code is that we named the field `teamId` in the `Event` type, and the API endpoint is expecting `team_id` instead.